Docs/source/pages/advanced-usage.md

```diff
@@ -496,6 +496,12 @@ Keep in mind that, the tags given in the constructor will choose one random free
     In order to ensure that the `dotnet test` command is executed multiple times concurrently within the same terminal add an `&` at the end of the command to run it in the background.
 ```
 
+```eval_rst
+.. note::
+
+    Make sure that your **product name are different** in case you started multiple instrumented apps on the **same device**, otherwise your tests might fail because they are using the same resources (ex. values saved to `PlayerPref`).
```

Comment:
```suggestion
    Make sure that your **product names are different** in case you started multiple instrumented apps on the **same device**, otherwise your tests might fail because they are using the same resources (ex. values saved to `PlayerPref`).
```